pkg/metrics/metrics.go

```diff
@@ -11,28 +9,43 @@ const (
 
 	sharesSubsystem = "openshift_csi_share"
 
-	mount          = "mount"
-	mountCountName = sharesSubsystem + separator + mount + separator + "total"
+	mount  = "mount"
+	failed = "failed"
+
+	mountCountName  = sharesSubsystem + separator + mount + separator + "total"
```

Comment:
I think the `requests_total` and `failure_totals` names that @jan--f has in #78 is better for these @alicerum

Reply:
ok assign yourself to that one @alicerum and give it an lgtm .... I'll then approve